app/Http/Controllers/Api/UsersController.php

```diff
@@ -53,10 +53,36 @@ public function __construct(Registrar $registrar, Manager $manager, UserReposito
      *
      * @return \Illuminate\Http\Response
      */
-    public function index()
+    public function index(UserRequest $request)
     {
-        // @TODO: temp for now, likely want to use repository and transform.
-        return User::all();
+        $user = User::find($request['id']);
```

Comment:
This isn't vital for merge but something to consider. In REST an `index` endpoint typically always returns a collection/array of items, in this case users, and allows for filtering it down _if_ parameters are passed.